app/services/force/request.rb

```diff
@@ -38,6 +38,8 @@ def cached_get(endpoint, params = nil, force = false)
         "running cached_get for #{endpoint} with force set to #{force_refresh}",
       )
       @cache.fetch(key, force: force_refresh, expires_in: expires_in) do
```

Comment:
question: I'm not following how the code changes in this PR will solve this issue:
> If there isn’t already a cache (i.e. the cache is expired or it’s a new deploy) then the caching doesn’t work

Can't we just pass in `true` for the `force:` option here? It's what triggers a cache write, [docs here](https://api.rubyonrails.org/v7.0.0/classes/ActiveSupport/Cache/Store.html#method-i-fetch)

Reply:
Ok interesting point about how force triggers a cache write, but I think the caching we are missing is in cache_single_listing, which is separate

I named the param cache_only to indicate "grab only what's in the cache" so we can check if we need to move forward and do the cache_single_listing calls

Reply:
I am still pondering on this because I can't figure out why the built in caching doesn't do what we need it to do - will get back to you after some investigation